account-kit/react/src/components/auth/card/eoa.tsx

```diff
@@ -9,7 +9,13 @@ import { useConnectEOA } from "../hooks/useConnectEOA.js";
 import { useWalletConnectAuthConfig } from "../hooks/useWalletConnectAuthConfig.js";
 import { CardContent } from "./content.js";
 import { ConnectionError } from "./error/connection-error.js";
-import { EOAWallets } from "./error/types.js";
+import { WalletIcon } from "./error/icons/wallet-icon.js";
+
+enum EOAWallets {
```

Comment:
is this still used? we shouldn't hardcode these because any number of things can show up for injected wallets